deployment/terraform/metrics.go

```diff
@@ -4,14 +4,20 @@
 package terraform
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"net/http"
 	"strings"
 
 	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
 
 	"github.com/mattermost/mattermost-server/v5/mlog"
 )
 
+const defaultGrafanaUsernamePass = "admin:admin"
```

Comment:
Should this come from `deployer.json`?

Reply:
Not really. This is the initial user/pass which is always fixed. The user needs to change the password in the first login.

Reply:
I was suggesting disabling that behaviour with `disable_initial_admin_creation` and set admin username/password as defined by the user.

https://grafana.com/docs/grafana/latest/installation/configuration/#security

Just a thought, especially since this instance is probably going to be accessed from the outside.

Reply:
I haven't given too much thought on security till now. Even prometheus is exposed to the world, and port 8067 too. We can do what you are suggesting for sure. But I want to take a comprehensive sweep of the security side of this once we are done with the MVP. At that time, we can take care of this amongst other things.

Reply:
Sounds like a plan :+1: